assets/lang/es_ES.ini

```diff
@@ -605,23 +649,41 @@ An.Right = Stick derecha
 An.Up = Stick arriba
 Analog limiter = Limitador analóg.
 Analog Stick = Stick analógico
+Arrow down = ลูกศรลง
```

Comment:
Thai striked again? It seem to be like this in all the .ini (didn't check them all tho') :)